### PR TITLE
Update manifest.in

### PR DESCRIPTION
**Pull request by OrkoHunter**
I noticed this while trying
```
python setup.py bdist_wheel
```

### CODE REVIEW

**Comment by OrkoHunter** [comment]
Can we have the first release candidate for `networkx-metis`? I see two open issues about changing the tests and making conda recipe. But in my opinion that doesn't come in the way of releasing it on pypi.

File: MANIFEST.in [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Wrapper files
-graft networkx
+graft nxmetis
 
 # Build files
-exclude _metis.c
+exclude nxlemon/_metis.c
```
Comment (SanketDG) [inline MANIFEST.in]: Did you mean `nxmetis`?

Comment (OrkoHunter) [inline MANIFEST.in]: Oh yeah. Sorry

**Comment by ysitu** [comment]
I will open an Issue for the specifics of the 1.0 release.

**Pull request merged**